src/experiments/logins/api.js

```diff
@@ -49,6 +49,20 @@ this.logins = class extends ExtensionAPI {
           //   - fooLogin = login as vanilla JS object
           //   - fooLoginInfo = login as nsILoginInfo
           //   - fooBag = (subset of) login fields as nsIPropertyBag
+          getLoginSavingEnabled(aHost) {
```

Comment:
I really dislike the `aFoo` naming convention. How about just calling this `origin`?

Reply:
Heh, I used aHost to match the upstream API, but I can change it if necessary